packages/examples/edgex/README.md

```diff
@@ -0,0 +1,15 @@
+# EdgexFoundry and Pulse integration through liota.
+
+Integrate EdgeXFoundry into Liota
+
+As the Open Interop Platform for the IoT Edge, EdgeX Foundry™, https://www.edgexfoundry.org/, is a vendor-neutral open source project building a common open framework for IoT edge computing. At the heart of the project is an interoperability framework hosted within a full hardware- and OS-agnostic reference software platform to enable an ecosystem of plug-and-play components that unifies the marketplace and accelerates the deployment of IoT solutions. So it’s worth explore how to integrate this into Liota.
+
+This directory includes:
+File 1: main.py
+File 2: http_server.py
+File 3: device_add_template.py
+
+## How to use
+```
+cp device_add_template.py /usr/lib/liota/packages/
```

Comment:
1. Also, add steps/link to install EdgeX or any configuration required to be done for EdgeX agent and device.
2. Add the step to load the edgex_device Package.